docs/tutorial/sandbox.md

```diff
@@ -0,0 +1,136 @@
+
+Unsandboxed renderers are not a problem for desktop applications that only display
+trusted code, but it makes Electron less secure than Chromium for displaying untrusted
+web content. For applications that require more security, sandboxed renderers are
+recommended.
+
+Sandboxed processes in Electron behave _mostly_ in the same way as Chromium's do, but
+Electron has a few additional quirks to consider.
+
+### Renderer processes
+
+When renderer processes in Electron are sandboxed, they behave in the same way as a
+regular Chrome renderer would. A sandboxed renderer won't be initialized in a Node.js
+environment, nor will it modify any default JavaScript APIs that behave differently
+by default in Electron (e.g. `window.open`).
+
+<!-- TODO(erickzhao): when we have a solid guide for IPC, link it here -->
+Therefore, when the sandbox is enabled, renderer processes can only make changes to the
+system by delegating tasks to the main process via inter-process communication (IPC).
+
+### Preload scripts
+
+In order to allow renderer processes to communicate with the main process, preload
+scripts attached to sandboxed renderers will still have a limited Node.js environment
+enabled. Node's `require` module is exposed, but will only be able to import a subset
+of Electron and Node's built-in modules:
+
+* `electron` (only renderer process modules)
+* [`events`](https://nodejs.org/api/events.html)
+* [`timers`](https://nodejs.org/api/timers.html)
+* [`url`](https://nodejs.org/api/url.html)
+
+In addition, the preload script also exposes certain Node.js primitives as globals:
+
+* [`Buffer`](https://nodejs.org/api/Buffer.html)
+* [`process`](https://nodejs.org/api/process.html)
+* [`clearImmediate`](https://nodejs.org/api/timers.html#timers_clearimmediate_immediate)
+* [`setImmediate`](https://nodejs.org/api/timers.html#timers_setimmediate_callback_args)
+
+Note that because the Node.js environment is still present in the `preload`, it is still
+possible to leak privileged APIs to untrusted code running in the renderer process unless
+[`contextIsolation`][contextIsolation] is enabled.
+
+### Configuring the sandbox
+
+In Electron, renderer sandboxing can be enabled on a per-process basis with
+the `sandbox: true` preference in the [`BrowserWindow`][browser-window] constructor.
+
+```js
+// main.js
+app.whenReady().then(() => {
+  const win = new BrowserWindow({
+    webPreferences: {
+      sandbox: true
+    }
+  })
+  win.loadURL('http://google.com')
+})
+```
+
+If you want to force sandboxing for all renderers, you can also use the
+[`app.enableSandbox`][enable-sandbox] API. Note that this API has to be called before the
+app's `ready` event.
+
+```js
+app.enableSandbox()
+app.whenReady().then(() => {
+  // no need to pass `sandbox: true` since `app.enableSandbox()` was called.
+  const win = new BrowserWindow()
+  win.loadURL('http://google.com')
+})
+```
+
+You can also disable Chromium's sandbox entirely with the [`--no-sandbox`][no-sandbox]
```

Comment:
Can we put this in smaller font 😉 , it's good info to share but we should make it clear somehow that folks shouldn't be doing this

Reply:
I hope ce2fad5 is enough emphasis!